doc/lua_api.txt

```diff
@@ -2793,6 +2793,8 @@ Call these functions only at load time!
 
 * `minetest.register_globalstep(func(dtime))`
     * Called every server step, usually interval of 0.1s
+* `minetest.register_on_mods_loaded(func())`
+    * Called just after mods have finished loading.
```

Comment:
Could do well with a more detailed information when exactly it is called:
* Called after mods have finished loading and before the media is cached or the aliases handled.